experimental-default-global-styles.json

```diff
@@ -1,7 +1,41 @@
 {
-	"color": {
-		"primary": "#52accc",
-		"background": "white",
-		"text": "black"
+	"global": {
+		"presets": {
```

Comment:
While adapting this to the new `theme.json` schema I thought it was good to have the core presets already presents, as fallbacks for when the theme doesn't declare any.